docs/troubleshooting.md

```diff
@@ -169,3 +169,13 @@ $ curl -X POST localhost:9411/api/v2/spans -H'Content-Type: application/json' -d
 
 No response means the request was successfully sent. You can also pass `-v` to
 the `curl` command to confirm.
+
+## APM - Infrastructure Correlation
+
+### Setup the APM Service - Infrastructure correlation metrics
+
+How do you setup the collector to capture and send the relevant data to show
+correlated infrastructure metrics in the APM service dashboards?
+
+Follow the steps outlined [here](apm-infra-correlation.md) and if you are still running into
+issues then reach out to [support](#gathering-support-information)
```

Comment:
A few suggested changes:

- Change title to "Set up the APM service - infrastructure correlation metrics".
- Change "setup" to "set up".
- Change the last sentence to "Follow the steps outlined here. If you are not able to set up the APM service after following the instructions, contact support."
- Change "here" to the name of the page you are linking to.

Reply:
@tmorrisonsfx for the last suggestion - 
I'm not sure how to construct that sentence without "here".
Follow the steps outlined <Enable APM Correlation> ?